### PR TITLE
Upgrade to laravel 11

### PR DESCRIPTION
**Pull request by yormy**
Upgrade composer to support laravel 11

### CODE REVIEW

**Comment by Henriquex25** [comment]
@rennokki ??

**Comment by vermaysha** [comment]
@rennokki please merge this pr

**Comment by yormy** [comment]
In the mean time you could use my fork https://github.com/yormy/laravel-eloquent-query-cache.git

File: composer.json [hunk 1]
```diff
@@ -36,13 +36,13 @@
...
         "test": "vendor/bin/phpunit"
     },
-    "require-dev": {
-        "chelout/laravel-relationship-events": "^1.5|^2.0",
-        "laravel/legacy-factories": "^1.3",
-        "livewire/livewire": "dev-master",
-        "mockery/mockery": "^1.5",
-        "orchestra/testbench": "^7.23|^8.1.1",
-        "phpunit/phpunit": "^9.5.25"
+    "require-dev1": {
```
[the remark this answers is not in the document]
Comment (yormy) [reply composer.json]: you are right.. fixed it...

**Comment by GORGDEVELOPER** [comment]
@rennokki Can you merge it please?

**Review by Smokietr — APPROVED** [review]

**Comment by lloricode** [comment]
this package mention here https://laracasts.com/series/small-laravel-packages-big-impact/episodes/12, but today can not install on latest laravel, hope someone check this PR

**Comment by iRaziul** [comment]
@rennokki Are you there? its been a while

**Comment by NebsterSK** [comment]
Any update on merging this, please?

**Comment by bernhardh** [comment]
Still no upgrade. This project/package is dead?

**Comment by iRaziul** [comment]
> Still no upgrade. This project/package is dead?

it seems like the author have no interest in keeping this package alive 🥲

**Comment by defser** [comment]
I would advise against this PR because of the missing changes in .github/workflows/ci.yml

Better option would be to fix and merge:
https://github.com/renoki-co/laravel-eloquent-query-cache/pull/213

**Comment by nicholaszuccarelli** [comment]
Any updates?

**Comment by oliverbj** [comment]
Any update or alternative to this?

**Comment by modrink** [comment]
Thank you @rennokki

**Pull request merged**